workshops/sprig_dodge/README.md

```diff
@@ -0,0 +1,511 @@
+@title: dodge_the_fireball
+@author: sam liu
+*/
+```
+
+Let's start programming!
+
+## 2. Making the sprites
+
+We have to tell Sprig what our characters, players, and objects are. This is possible with sprites.
+
+### a) Defining the sprites
+
+In our game, we'll be using two sprites: the player and the obstacles.
+Each sprite has a key, this is a single character that tells Sprig which sprite is which.
+
+```js
+const player = "p";
+const obstacle = "o";
+```
+
+> `p` and `o` are assigned to our sprites, `player` and `obstacle`, respectively. 
+
+### b) Making art for the sprites
+
+Add the following to your code.
+
+```js
+setLegend(
+  [obstacle, bitmap`...`],
+  [player, bitmap`...`]
+    
+);
+```
+
+> `setLegend()` will tell Sprig what image to display for each sprite. <br> We put the obstacle on top as we want it to display above the player.
+
+Click on the green `bitmap` text to open the pixel editor. Select colors and tools on the right to draw your sprites in the 16x16 area. Be creative with your design.
+
+> The bitmap is stored as a string. You can click on the arrow next to the line number to expand or hide it. <br> Each character represents a pixel and its color.
+
+<details>
+<summary>Here's what I came up with.</summary>
+
+```js
+setLegend(
+    [player, bitmap`
+................
+................
+................
+.....00000......
+....0.....0.....
+....0.0.0.0.....
+....0.....0.....
+....0.000.0.....
+....0.....0.....
+.....00000......
+.......0........
+.....00000......
+.......0........
+.......0........
+......0.0.......
+.....0...0......`],
+    [obstacle, bitmap`
+.......66.......
+........6.......
+.....66.6.6.....
+....66.66.66....
+....666666.6....
+...6699999966...
+...69999999966..
+..669999999996..
+..669933339996..
+..699333333996..
+..699333333996..
+...69333333996..
+...69333333996..
+...6993333996...
+....66999996....
+.....666666.....`]
+);
+```
+
+> Your design probably will be different. <br> I tried my best, my art skills are not the best.
+
+</details>
+
+Hit the run button to load the legend. Nothing will show up yet but This will allow us to make our map in the next step.
+
+## 3. Making the level
+
+Levels are the easiest way to tell Sprig what to display on the screen. We'll only be having one level.
+
+The following code can be used to make and set the level map.
+
+```js
+setMap(map`.`);
+```
+
+Similar to sprites, click on the green `map` text to open up the map editor. Use the `+` and `-` signs on the edges to expand ot shrink the map. We want to create a 8x8 map.
+
+Our map will only contain the player. We will write code later to spawn in the obstacles.
+
+It should look something like this.
+![TODO: Add a img of the map editor]()
+
+The code after expanding should be similar to this.
+
+```js
+setMap(map`
+........
+........
+........
+........
+........
+........
+........
+...p....`);
+```
+
+> Similar to the sprites, each character represents a sprite. <br> The `.` represents an empty tile.
```

Comment:
```suggestion
> Similar to the sprites, each character represents a sprite. The `.` represents an empty tile.
```

Can you remove all other `<br>`s, they're a bit unreliable in `@hackclub/markdown`

Reply:
Is there a way to have multiple lines in `>` without it spliting into seperate sections of quotes?

Reply:
Seems like it was the problem of my VSCode markdown preview extension, it works fine here on gh.